/**:
  ros__parameters:
    ir_width: 640
    ir_height: 480
    ir_fps: 30
    ir_frame_id: "camera_ir_frame"
    ir_optical_frame_id: "camera_ir_optical_frame"
    enable_ir: true
    depth_width: 640
    depth_height: 480
    depth_fps: 30
    depth_frame_id: "camera_depth_frame"
    depth_optical_frame_id: "camera_depth_optical_frame"
    enable_depth: true
    publish_tf: true
    tf_publish_rate: 10.0
    reconnect_timeout: 6.0
    depth_registration: true
    serial_number: ""
    camera_link_frame_id: "camera_link"
